README.md

```diff
@@ -121,6 +121,146 @@ var me = $rdf.sym('http://www.w3.org/People/Berners-Lee/card#i');
+
+[here]: https://www.w3.org/TR/rdf-sparql-query/
+
+.
+
+The first step in querying data storerd in your Solid Pod, is to fetch the data from your Pod into your local store. This is performed by the function "loadFromUrl". This function takes in a url from where your Pod can be reached and the variable that defines your local store. 
+
+The next step is call "prepare" in order to convert a SPARQL query string into a query object that can be used to run the query. After the query is prepared, execute the query by passing in the created query object and the local store you fetched your pod into. This will return an arrray of results based on what query the query returns from your local store. 
+
+```javascript
+import $rdf from "rdflib";
+const query = "select distinct ?s ?p ?o where {  ?s ?p ?o. }";
+const store = $rdf.graph();
+
+// Loads the data from a URL into the local store
+const loadFromUrl = (url, store) => {
+  return new Promise((resolve, reject) => {
+    let fetcher = new $rdf.Fetcher(store);
+    try {
+      fetcher.load(url).then(response => {
+        resolve(response.responseText);
+      });
+    } catch (err) {
+      reject(err);
+    }
+  });
+};
+
+// Prepares a query by converting SPARQL into a Solid query
+const prepare = (qryStr, store) => {
+  return new Promise((resolve, reject) => {
+    try {
+      let query = $rdf.SPARQLToQuery(qryStr, false, store);
+      resolve(query);
+    } catch (err) {
+      reject(err);
+    }
+  });
+};
+
+// Puts query results into an array according to the projection
+const rowHandler = (wanted, results) => {
+  const row = {};
+  for (var r in results) {
+    let found = false;
+    let got = r.replace(/^\?/, "");
+    if (wanted.length) {
+      for (var w in wanted) {
+        if (got === wanted[w].label) {
+          found = true;
+          continue;
+        }
+      }
+      if (!found) continue;
+    }
+    row[got] = results[r].value;
+  }
+  return row;
+};
+
+// Executes a query on the local store
+const execute = (qry, store) => {
+  return new Promise((resolve, reject) => {
+    const wanted = qry.vars;
+    const resultAry = [];
+    store.query(
+      qry,
+      results => {
+        if (typeof results === "undefined") {
+          reject("No results.");
+        } else {
+          let row = rowHandler(wanted, results);
+          if (row) resultAry.push(row);
+        }
+      },
+      {},
+      () => {
+        resolve(resultAry);
+      }
+    );
+  });
+};
+
+async function getData() {
+  loadFromUrl(url, store).then(() =>
+    prepare(query, store).then(qry =>
+      execute(qry, store).then(results => {
+        // Do somthing with results here
+      })
+    )
+  );
+}
+```
+
+
+
+## Uploading data to POD
+
+```javascript
+const $rdf = require("rdflib");
+const store = $rdf.graph();
+var updater = new $rdf.UpdateManager(store);
+
+
+// Address of the named node that will store you uploaded data
```

Comment:
`store you uploaded`
->
`store your uploaded`

Reply:
Thank you for the edits!